README.md

```diff
@@ -12,14 +14,20 @@ Alternatively, you can get the path with `get_curator_path`.
 
 ## Building the package:
 
-Make sure you edit the hash in `testdir/test_basic.py`, `curatorbin/__init__.py` and `evergreen.yml`. 
-You can use `./changeover.sh ${OLD_HASH} ${NEW_HASH}` for this.
+You can use `./updateFromCurator.sh <major|minor|patch>` for this.
+
+For example, `./updateFromCurator.sh minor` will update the minor version, which would turn `1.2.3` to `1.3.0` in setup.py.
 
-Also, increment the version in the `setup.py` and `evergreen.yml` files. 
+This will be used to increment the version in the `setup.py` and `evergreen.yml` files. 
+It also replaces with the latest curator hash the hash in `testdir/test_basic.py`, `curatorbin/__init__.py` and `evergreen.yml`. 
 
-Then, run a mainline or patch build and make sure to use the `publish-to-testpypi` evergreen task.
+
+After running `updateFromCurator.sh`, commit and  run a mainline or patch build.
```

Comment:
nit: remove random whitespace ("commit and run").

Reply:
I wouldn't run a mainline build to test the update since that requiress committing and pushing your change to the main branch.